hrcareer/js/src/hrcareer/app.js

```diff
@@ -1,18 +1,16 @@
 /* eslint-env amd */
 
-define(function () {
-  'use strict';
+(function (CRM) {
```

Comment:
not a major point, but you might consider
```js
(function ($) {

}(CRM.$));
```

Reply:
Done.